From 830164e6deb8086257fae411c0fbf17f42868fff Mon Sep 17 00:00:00 2001
Message-Id: <830164e6deb8086257fae411c0fbf17f42868fff.1659372189.git.thomas@gllm.fr>
In-Reply-To: <5b01e96eabd14521cfeb33b5ac11495b77f894cc.1659372189.git.thomas@gllm.fr>
References: <5b01e96eabd14521cfeb33b5ac11495b77f894cc.1659372189.git.thomas@gllm.fr>
From: Thomas Guillem <thomas@gllm.fr>
Date: Mon, 1 Aug 2022 17:45:26 +0200
Subject: [PATCH 19/19] WIP: ci: android: use last image and last build script

Last image that contains the NDK25.

WIP since the new build script is not yet merged in videolan repo.
---
 extras/ci/gitlab-ci.yml | 10 ++++++----
 1 file changed, 6 insertions(+), 4 deletions(-)

diff --git a/extras/ci/gitlab-ci.yml b/extras/ci/gitlab-ci.yml
index 53f3c60c55..9e37d77ee7 100644
--- a/extras/ci/gitlab-ci.yml
+++ b/extras/ci/gitlab-ci.yml
@@ -24,7 +24,7 @@ variables:
     VLC_WIN_LLVM_IMAGE: registry.videolan.org/vlc-debian-llvm-mingw:20220705072259
     VLC_UWP_LLVM_IMAGE: registry.videolan.org/vlc-debian-llvm-uwp:20220713124227
     VLC_DEBIAN_IMAGE: registry.videolan.org/vlc-debian-unstable:20210803114245
-    VLC_ANDROID_IMAGE: registry.videolan.org/vlc-debian-android:20220505164734
+    VLC_ANDROID_IMAGE: registry.videolan.org/vlc-debian-android:20220728132611
     VLC_SNAP_IMAGE: registry.videolan.org/vlc-ubuntu-focal:20220505174948
     VLC_RASPBIAN_IMAGE: registry.videolan.org/vlc-ubuntu-raspberry:20220505175523
     VLC_WASM_EMSCRIPTEN: registry.videolan.org/vlc-debian-wasm-emscripten:20210915101305
@@ -447,11 +447,13 @@ tvos-arm64:
     image:
         name: $VLC_ANDROID_IMAGE
     script: |
-        wget https://code.videolan.org/videolan/vlc-android/raw/fd4d934ad31d37f3eeaa834214ca16eb0d4ecb6a/buildsystem/compile-libvlc.sh
+        #git clone https://code.videolan.org/videolan/libvlcjni.git
+        git clone https://code.videolan.org/tguillem/libvlcjni.git
+        (cd libvlcjni && git checkout 381ee2f470462b52638af08373895547bac385d5)
         if [ -n "$VLC_PREBUILT_CONTRIBS_URL" ]; then
-            /bin/sh ./compile-libvlc.sh -a $ANDROID_ARCH --with-prebuilt-contribs
+            /bin/sh ./libvlcjni/buildsysten/compile-libvlc.sh -a $ANDROID_ARCH --with-prebuilt-contribs
         else
-            /bin/sh ./compile-libvlc.sh -a $ANDROID_ARCH --package-contribs
+            /bin/sh ./libvlcjni/buildsystem/compile-libvlc.sh -a $ANDROID_ARCH --package-contribs
         fi
 
 android-arm:
-- 
2.35.1

